content/starting-team/why.md

```diff
@@ -11,6 +11,6 @@ It’s exciting to start something new. It’s also a big challenge — designin
 
 If you can get the right roles in the team from the start you will build momentum and be able to start delivering and iterating quicker.
 
-The team needs the right combination of roles to deliver services that meet the needs of users.
+The team needs the right combination of roles to build services that meet the needs of users.
 
 The people in the team also need to be open to collaborating and working in new ways.
```

Comment:
Whoop! I missed this one we should change it to `open to working together and in new ways`.

There will be a few more `collaborating` I'll need to fix.